src/Service/Resolvers/BaseSqlQueryBuilder.cs

```diff
@@ -23,6 +23,11 @@ public abstract class BaseSqlQueryBuilder
         public const string SCHEMA_NAME_PARAM = "schemaName";
         public const string TABLE_NAME_PARAM = "tableName";
 
+        /// <summary>
+        /// Predicate added to the query when no valid predicates exist.
```

Comment:
nit: if no 'valid' predicates exist, I would imagine the request should fail. Maybe this should say "when no other predicates exist"